ross/fluid_flow/fluid_flow.py

```diff
@@ -0,0 +1,400 @@
+            at some point. It is useful in case the user wants to see one graphic alongside another.
+        """
+        plt.figure(self.plot_counter)
+        self.plot_counter += 1
+        x = np.zeros(self.nz)
+        y_re = np.zeros(self.nz)
+        y_ri = np.zeros(self.nz)
+        for i in range(0, self.nz):
+            x[i] = i * self.dz
+            y_re[i] = self.re[i][theta]
+            y_ri[i] = self.ri[i][theta]
+        plt.plot(x, y_re, 'r')
+        plt.plot(x, y_ri, 'b')
+        plt.title('Shapes of stator and rotor along Z; Theta='+str(theta))
+        plt.xlabel('Points along Z')
+        plt.ylabel('Radial direction')
+        plt.show(block=show_immediately)
+        if show_immediately:
+            plt.close('all')
+
+    def plot_pressure_theta_cylindrical(self, z=0, show_immediately=True):
+        """This function assembles cylindrical pressure graphic in the theta direction for a given z.
+        Parameters
+        ----------
+        z: int
+            The distance along z-axis to be considered.
+        show_immediately: bool
+            If True, immediately plots the graphic. Otherwise, the user should call plt.show()
+            at some point. It is useful in case the user wants to see one graphic alongside another.
+        """
+        if not self.pressure_matrix_available:
+            sys.exit('Must calculate the pressure matrix.'
+                     'Try calling calculate_pressure_matrix first.')
+        r = np.arange(0, self.radius_stator + 0.0001, (
+            self.radius_stator - self.radius_rotor)/self.nradius
+            )
+        theta = np.arange(0, 2*np.pi + self.dtheta/2, self.dtheta)
+
+        pressure_along_theta = np.zeros(self.ntheta)
+        for i in range(0, self.ntheta):
+            pressure_along_theta[i] = self.p_mat[0][i]
+
+        min_pressure = np.amin(pressure_along_theta)
+
+        r_matrix, theta_matrix = np.meshgrid(r, theta)
+        z_matrix = np.zeros((theta.size, r.size))
+        inner_radius_list = np.zeros(self.ntheta)
+        pressure_list = np.zeros((theta.size, r.size))
+        for i in range(0, theta.size):
+            inner_radius = np.sqrt(self.xri[z][i] * self.xri[z][i] + self.yri[z][i] * self.yri[z][i])
+            inner_radius_list[i] = inner_radius
+            for j in range(0, r.size):
+                if r_matrix[i][j] < inner_radius:
+                    continue
+                pressure_list[i][j] = pressure_along_theta[i]
+                z_matrix[i][j] = pressure_along_theta[i] - min_pressure + 0.01
+        fig, ax = plt.subplots(subplot_kw=dict(projection='polar'))
+        self.plot_counter += 1
+        ax.contourf(theta_matrix, r_matrix, z_matrix, cmap='coolwarm')
+        plt.title('Pressure along Theta; Z='+str(z))
+        plt.show(block=show_immediately)
+        if show_immediately:
+            plt.close('all')
+
+    def plot_pressure_theta(self, z=0, show_immediately=True):
```

Comment:
When using a function to create a plot with matplotlib, it is better to make the axes an argument:
```python
def foo(ax=None):
    # if axes is not given get the one from the current figure
    if ax is None:
        ax = plt.gca()
    .
    .
    ax.plot(...)
    .
    .
    return ax
```
For more details about matplotlib objects (figure, axes, axis etc.) refer to https://matplotlib.org/tutorials/introductory/usage.html

Reply:
We will soon change from matplotlib to bokeh, so I think we can let them like this for now. What do you think?